### PR TITLE
treewide: Reformat with ipt

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

**Comment by Garados007** [comment]
@t-b You just used `ipt format` and no `ipt lint`. Right?

**Comment by t-b** [comment]
> @t-b You just used `ipt format` and no `ipt lint`. Right?

Yes!

File: tests/test-tracing_instrumented.ipf
[the remark this answers is not in the document]
Comment (t-b) [reply tests/test-tracing_instrumented.ipf:1]: Yes, tests/tracing.ipf and tests/tracing_instrumented.ipf should not be formatted. This then also avoids hitting IPT bug #302.

**Comment by t-b** [comment]
> Also how about adding a `.git-blame-ignore-revs` file like in MIES? The rest looks good.

Done.

**Review by Garados007 — APPROVED** [review]

**Pull request merged**